pkg/resources/pods.go

```diff
@@ -81,8 +81,10 @@ func (pc PodAccessor) ReadyCount() (int, error) {
 		// Cheap to check that the pod is running.
 		if p.Status.Phase == corev1.PodRunning && p.DeletionTimestamp == nil {
 			for _, cond := range p.Status.Conditions {
-				if cond.Type == corev1.PodReady && cond.Status == corev1.ConditionTrue {
-					ret++
+				if cond.Type == corev1.PodReady {
```

Comment:
why not:

```
if p.Status.getConfition(corev1.PodReady).IsTrue() {
    ret++
}
```

Reply:
I don't think corev1 types have that interface?

Reply:
oh I forgot where we are. I wish some of our duck ideas were in upstream

Reply:
/lgtm